sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/WideSchemaBenchmark.scala

```diff
@@ -54,8 +54,11 @@ class WideSchemaBenchmark extends SparkFunSuite with BeforeAndAfterEach {
   }
 
   override def afterAll() {
-    super.afterAll()
-    out.close()
+    try {
+      super.afterAll()
+    } finally {
+      out.close()
```

Comment:
Is it correct in this test suite context? We usually put `super.afterAll()` into `finally`.

Reply:
IMHO this is clearly an error and we should swap it